core/trino-main/src/main/java/io/trino/operator/ExchangeOperator.java

```diff
@@ -41,24 +57,27 @@
     {
         private final int operatorId;
         private final PlanNodeId sourceId;
-        private final DirectExchangeClientSupplier exchangeClientSupplier;
+        private final DirectExchangeClientSupplier directExchangeClientSupplier;
```

Comment:
this rename should probably be part of the commit which renames `ExchangeClient` to `DirectExchangeClient`